MAINTAINERS.md

```diff
@@ -45,3 +45,10 @@ To become a maintainer, open a pull request to this list.
 * [Fletcher Nichol](https://github.com/fnichol)
 * [Romain Sertelon](https://github.com/rsertelon)
 * [Scott Hain](https://github.com/scotthain)
+
+## Approvers
```

Comment:
@eeyun, @stevendanna:  I've added this "approvers" section and adding myself along with Graham to the list.  I've also removed the term 'veteran' in favour of the 'approver' member, which is defined in the Chef Project Member handbook.  I've left the 'Active Members' section as-is (I didn't change this to be 'Approvers').  If 'Active Members' wish, they can raise a PR to get their names moved into the Approvers list.

Reply:
Unless you (and @predominant) want me to make any more changes in the next couple days, I'll plan on merging this by the end of the week.  Thanks all.